src/core/js/views/drawerView.js

```diff
@@ -86,7 +86,14 @@ define([
             this._isCustomViewVisible = true;
             Adapt.trigger('drawer:empty');
             this.showDrawer();
+
             this.$('.drawer-holder').html(view);
+
+            var model = this.collection.findWhere({
```

Comment:
@barryo-learningpool, I think if we want to do this properly, we'll need to update all the plugins that use Drawer to pass their actual view to `Adapt.drawer.triggerCustomView()` rather than their view's element.

For example, in [Resources](https://github.com/adaptlearning/adapt-contrib-resources/blob/19db7d410e96b5462108d3054f010485bb7c47c8/js/adapt-contrib-resources.js#L14-L17), removing the `.$el` from:
```js
Adapt.drawer.triggerCustomView(new ResourcesView({
    model: resourcesModel,
    collection: resourcesCollection
}).$el);
```
Most plugins will then already have their model available (PLP will need one adding in the same vein as [Resources](https://github.com/adaptlearning/adapt-contrib-resources/blob/19db7d410e96b5462108d3054f010485bb7c47c8/js/adapt-contrib-resources.js#L10)).

To keep backwards compatibility, in this PR we'll need to change [this line](https://github.com/barryo-learningpool/adapt_framework/blob/07a3722783e015f4163eb73b173e5a7dbb446676/src/core/js/views/drawerView.js#L90) to
```js
this.$('.drawer-holder').html(view.$el || view);
```
and [`model`](https://github.com/barryo-learningpool/adapt_framework/blob/07a3722783e015f4163eb73b173e5a7dbb446676/src/core/js/views/drawerView.js#L92-L94) will become
```js
var model = view.model;
```
Happy with this approach?